### PR TITLE
docs: Update Card and DropDown to use Anchor components in examples

### PR DESCRIPTION
**Pull request by zero0Halo**
chore(typings): added Card, InfoOutline, List, ListDivider & ListTitle

fix(MoreInfo): using DropDown for its example rather than homebrew

fix(MyList) using List for its example rather than homebrew

docs(card and dropdown): Updated code blocks based on the above changes

chore: prettier

**Before submitting a pull request,** please make sure the following is done:

I have done **all** of the following:

- [ ] Added a top level class to all my components `'.anchor-[COMPONENT NAME]'`.
- [x] Used [conventional commits](https://www.conventionalcommits.org) for all work.
- [ ] Tested my solution on Mobile & Tablet.
- [ ] Wrote [unit tests](https://jestjs.io/docs/en/getting-started) for states and all behavior (`npm test`) and passed coverage thresholds.
- [ ] Updated snapshots for all permutations (`npm test -- -u`).
- [ ] Accounted for hover, focus, blur, visited, & error states because they are not edge cases.
- [ ] Created TODOs for known edge cases.
- [ ] Documented all of my changes (inline & doc site).
- [ ] Made sure that all accessibility errors are resolved.
- [ ] Added [stories](https://storybook.js.org/docs/basics/introduction/) with knobs for all possible configurations.
- [x] De-linted and ran [prettier](https://github.com/prettier/prettier) (`npm run pretty`) on my code.
- [ ] Added name to OWNERS file for all new components
- [ ] If adding a new component, add its export to the rollup config
- [ ] package.json version is bumped (if necessary)

---------
**Outline your feature or bug-fix below**

My code does...

### CODE REVIEW

File: docs/src/typings.d.ts [hunk 1]
```diff
@@ -11,6 +11,7 @@ declare module '@retailmenot/anchor' {
   const AutoComplete: any;
   const Badge: any;
   const Button: any;
+  const Card: any;
```
Comment (sharkparty) [inline docs/src/typings.d.ts]: We actually ship anchor with its own typings file so this shouldn't be needed so long as the client is on the latest version of Anchor. Also, typing anything `any` isn't a great idea. It's good for accelerating against a tough deadline, but it's not a sustainable practice.

**Review by sharkparty — CHANGES_REQUESTED** [review]
Let's try and figure out the typings issue.

File: docs/src/typings.d.ts (hunk 1 above)
Comment (zero0Halo) [reply docs/src/typings.d.ts]: I'll look into this. If we're shipping the types, it's really odd they aren't coming through. Agreed about `any`, this list has just been building and building. Need to kill the trend.

Comment (zero0Halo) [reply docs/src/typings.d.ts]: For posterity, @sharkparty discovered that removing the Anchor type definitions in the repo solves the issue. Most likely these were given priority over the definitions in the Anchor module.

**Review by sharkparty — APPROVED** [review]
LGTM

**Pull request merged**